otj-metrics-core/src/main/java/com/opentable/metrics/MetricSetBuilder.java

```diff
@@ -138,6 +140,15 @@ public AtomicLong longGauge(String name) {
         return create(name, AtomicLongGauge::new);
     }
 
+    /**
+     * Create a histogram
+     * @param name the name of the histogram to create
+     * @return a new histogram
+     */
+    public Histogram histogram(String name) {
```

Comment:
Document as using this resource and provide another overloaded method to specify. Otherwise looks good